<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WKSP00075683" ignored="0" name="Test" tests="2" failures="2" timestamp="06 Feb 2017 16:30:44 GMT" time="11.722" errors="0">
  <testcase name="testEasy" time="0.854" classname="MAX.SeleniumHQ">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
	at MAX.SeleniumHQ.testEasy(SeleniumHQ.java:23)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testEasy -->
  <testcase name="valid_UserCredential" time="5.756" classname="MAX.SeleniumHQJUNIT">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at MAX.SeleniumHQJUNIT.valid_UserCredential(SeleniumHQJUNIT.java:32)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- valid_UserCredential -->
</testsuite> <!-- Test -->
